alkos/kernel/CMakeLists.txt

```diff
@@ -11,8 +11,8 @@ include_directories(test)
 # Define ARCH_DIR
 set(ARCH_DIR "${CMAKE_CURRENT_SOURCE_DIR}/arch/${ARCH}")
 
-# Additional test definitions
-# add_compile_definitions(ALKOS_TEST=0)
+# Additional test definitions - refer to test/tester.hpp file for value specifics
+# add_compile_definitions(ALKOS_TEST=1)
```

Comment:
I have the fact that in every pr this gets randomly changed. I mean, at least create a target that has this switched on, and one that has it switched off

Reply:
Reverted the change, we should get rid of it asap